Library/Output Templates/HTML - NPC Card.html

```diff
@@ -23,6 +23,7 @@
 			border: 2px solid black;
 			margin: 0.05in;
 			text-overflow: ellipsis;
+			overflow: hidden;
 			overflow-y: hidden;
```

Comment:
Should be able to remove the `overflow-y`, since `overflow` is already set to that, right?